homeassistant/components/enphase_envoy/quality_scale.yaml

```diff
@@ -0,0 +1,96 @@
+rules:
+  # Bronze
+  action-setup:
+    status: done
+    comment: only actions implemented are platform native ones.
+  appropriate-polling:
+    status: done
+    comment: fixed 1 minute cycle based on Enphase Envoy device characteristics
+  brands: done
+  common-modules: done
```

Comment:
That'll be a todo with description.

Reply:
Ok, this one sounds more like a regular code improvement rather than not meeting the common module quality scale? Does that make this a done or todo, or a general recommendation?

Reply:
An optional sidequest